apps/todoapp/src/app/todos/services/auth.service.ts

```diff
@@ -21,12 +23,12 @@ export class AuthService {
           // User is authenticated, proceed with your application
           console.log('User is authenticated:', user);
           this.user$.next(user as DiscordUser);
-        },
-        (error) => {
-          // 403 response, user is not authenticated
-          console.error('User is not authenticated:', error);
-          this.router.navigate(['/login']);
         }
+        // (error) => {
```

Comment:
Remove commented code.

Reply:
we need that for now but maybe we can get rid of it in the future when we make the authguard to take care of. I will explain the problem in our next sync. :)